ui/common/src/commonMain/kotlin/social/androiddev/common/composables/Spacers.kt

```diff
@@ -0,0 +1,64 @@
+/*
+ * This file is part of MastodonX.
+ *
+ * MastodonX is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+ *
+ * MastodonX is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with MastodonX. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package social.androiddev.common.composables
+
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.RowScope
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun RowScope.WeightSpacer(modifier: Modifier = Modifier) {
+    Spacer(modifier = modifier.weight(1f))
+}
+
+@Composable
+fun ColumnScope.WeightSpacer(modifier: Modifier = Modifier) {
+    Spacer(modifier = modifier.weight(1f))
+}
+
+@Composable
+fun XSmallSpacer(modifier: Modifier = Modifier) {
```

Comment:
I'd recommend adding a doc comment with the actual spacing size in it to all of these.

```kotlin
/**
 * Size: `2.dp`
 */
@Composable
fun XSmallSpacer(...) {
  // ...
}
```

Reply:
Agreed, that'd come useful 👍

Reply:
> [#66 (comment)](https://github.com/AndroidDev-social/MastodonCompose/pull/66#discussion_r1026354947)

I'm not sure what you're trying to say with this link? Using pixel dimensions is not good of course, but DP are totally fine (which is what you have here anyway).  Admittedly, I haven't worked very much with Compose Desktop, but I have used `dp` dimension values when I did.

Reply:
This PR is a first approximation so I hardcoded all the dimensions.

Dimensions (like paddings) must be defined in the Theme in order to support all kind of devices/platforms.
So, we shouldn't put this kind of comments because they can be a lie :)